src/os/shared/osapi-idmap.c

```diff
@@ -526,7 +526,7 @@ static int32 OS_ObjectIdFindNext(uint32 idtype, uint32 *array_index, OS_common_r
 
    if(return_code == OS_SUCCESS)
    {
-       return_code = OS_ObjectIdMap(idtype, idvalue, &obj->active_id);
+       return_code = OS_ObjectIdMap(idtype, idvalue, &obj->active_id); // lgtm[cpp/uninitialized-local]
```

Comment:
All of these are false alarms but could probably be avoid by setting both obj = NULL and local_id = 0 at the start of the function.